docs/subnets/customize-a-subnet.md

```diff
@@ -274,6 +272,21 @@ If you call `readAllowList(addr)` then you can read the current role of `addr`,
 
 WARNING: if you remove all of the admins from the allow list, it will no longer be possible to update the allow list without modifying the subnet-evm to schedule a network upgrade.
 
+#### Trustless Mode
+
+It's possible to enable this precompile without an admin address. In this mode you can define a list of addresses that are allowed to deploy contracts. This list is immutable and can't be changed. This mode is useful for networks that don't need a trusted party to manage the allow list. To enable this mode, you need to specify addresses in `enabledAddresses` field in your genesis or upgrade file:
```

Comment:
Can we replace the blurb `This list is immutable and can't be changed` with `This list requires an admin address to modify it or a network upgrade. If there are no admin addresses, this list can only be modified through a network upgrade` throughout this file.